.github/workflows/deploy.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         environment_url: https://npmjs.com
         job_status: ${{job.status}}
         slack_token: ${{secrets.NPM_ROBOT_SLACK_TOKEN}}
-        slack_channel: npm-ops
+        slack_channel: npm-test-notifications
```

Comment:
Need to revert that or not?

Reply:
i though maybe for this one we doesn't need that? like there are no real production deployments going on, or do you think its useful to have in npm-ops?

Reply:
Lets keep that in npm-test-notifications for now, as this deployment is actually just a test.

Reply:
👍